share/dkg/dkg.go

```diff
@@ -529,6 +548,12 @@ func (d *DistKeyGenerator) ProcessResponses(bundles []*ResponseBundle) (*Result,
 			continue
 		}
 
+		if bytes.Compare(bundle.SessionID, d.c.Nonce) != 0 {
+			// XXX for the moment we continue,
+			// TODO: fix it with a proper eviction list of share holders
```

Comment:
worth doing in this PR?

Reply:
:) https://github.com/drand/kyber/pull/14